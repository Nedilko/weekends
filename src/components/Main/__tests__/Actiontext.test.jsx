import Actiontext from '../ActionText'
import { render, screen } from '@utils/test-utils'

describe('Actiontext', () => {
  it('should render', () => {
    render(<Actiontext />)
    const title = screen.getByRole('heading')
    expect(title).toBeInTheDocument()
    expect(title).toMatchSnapshot()
  })
})
