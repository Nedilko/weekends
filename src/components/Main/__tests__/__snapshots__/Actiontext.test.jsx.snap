// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actiontext should render heading 1`] = `
<h2
  class="text-center text-3xl font-thin uppercase"
>
  Time left to weekends
</h2>
`;
